lib/runcible/response.rb

```diff
@@ -23,6 +23,10 @@ def body
       @parsed_body
     end
 
+    def to_hash
+      self.parsed_body.try(:to_hash)
```

Comment:
If you can verify that this response is consistent with what the old response returned when parsed_body was nil then ack for this. I am wondering if old response returned ``{}`` or just empty string or nil.